xline/src/storage/kvstore.rs

```diff
@@ -237,7 +237,46 @@ impl KvStoreInner {
             count: kvs.len().cast(),
             ..RangeResponse::default()
         };
-        response.kvs = kvs;
+        if !req.count_only {
+            match (req.sort_target(), req.sort_order()) {
```

Comment:
use req.sort_target/req.sort_order directly?

Reply:
Use them directly will match the i32 type, which works but is hard for humans to read.